tests/e2e/standard/enketo/pregnancy-visit.wdio-spec.js

```diff
@@ -0,0 +1,87 @@
+const moment = require('moment');
+const utils = require('../../../utils');
+const gatewayApiUtils = require('../../../gateway-api.utils');
+const loginPage = require('../../../page-objects/login/login.wdio.page');
+const commonPage = require('../../../page-objects/common/common.wdio.page');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const personFactory = require('../../../factories/cht/contacts/person');
+const userFactory = require('../../../factories/cht/users/users');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+const reportsPage = require('../../../page-objects/reports/reports.wdio.page');
+const genericForm = require('../../../page-objects/forms/generic-form.wdio.page');
+const pregnancyVisitForm = require('../../../page-objects/standard/forms/pregnancy-visit.wdio.page');
+
+describe('Pregnancy Visit', () => {
+  const places = placeFactory.generateHierarchy();
+  const healthCenter = places.find(place => place.type === 'health_center');
+  const user = userFactory.build({ place: healthCenter._id, roles: ['district_admin'] });
+  const pregnantWoman = personFactory.build({ patient_id: '12345', 
+    date_of_birth: moment().subtract(25, 'years').format('YYYY-MM-DD'), 
+    parent: {_id: healthCenter._id, parent: healthCenter.parent} });
```

Comment:
```suggestion
  const pregnantWoman = personFactory.build({
    patient_id: '12345',
    date_of_birth: moment().subtract(25, 'years').format('YYYY-MM-DD'), 
    parent: { _id: healthCenter._id, parent: healthCenter.parent }
  });
```
Minor, but is a lot easier to read if you format it a bit.